docs/user_guide/service_config.md

```diff
@@ -402,8 +402,8 @@ yunikornDefaults:
               value: namespace
               create: true
           queues:
-        - name: root
-          submitacl: '*'
+          - name: root
+            submitacl: '*'
```

Comment:
Looks like we are using the `do indent` style([Link](https://github.com/prettier/prettier/issues/4723)) in other places.  
So I think we should change it to below:
```
  queues:
    - name: root
    submitacl: '*'
```

Reply:
For example:
- https://yunikorn.apache.org/docs/user_guide/queue_config#child-template

Reply:
I think we have adopted the "whatever" indenting sequences attitude. That means either indenting or not indenting individual block sequences is OK.
I personally prefer this indentation:
```
queues:
  - name: root
    submitacl: '*'
```